src/screens/Inventory/components/PurchaseRequestCard.tsx

```diff
@@ -62,20 +79,25 @@ export const getPurchaseRequestStatusIcon = (status: PurchaseRequestStatus, size
 };
 
 function PurchaseRequestCard(props: PurchaseRequestCardProps) {
-  const { classes, purchaseRequest } = props;
+  const { classes, purchaseRequest, showSnackbarCallback } = props;
   const [status, setStatus] = useState(purchaseRequest.status);
   const product =
     useSelector((state: RootState) =>
       selectProductById(state, selectCurrentSiteInventoryById(state, purchaseRequest.inventoryId)?.productId || ''),
     ) || EMPTY_PRODUCT;
   const userId = useSelector(selectCurrentUserId);
   const userIsAdmin = useSelector(selectCurrentUserIsAdmin);
+  const isOnline = useSelector(selectIsOnline);
 
   const handleSubmitReview = (approved: boolean) => {
-    const newStatus = approved ? PurchaseRequestStatus.APPROVED : PurchaseRequestStatus.DENIED;
-    setStatus(newStatus);
-    purchaseRequest.status = newStatus;
-    reviewPurchaseRequest(purchaseRequest, approved, userId);
+    if (!isOnline) {
+      showSnackbarCallback();
+    } else {
+      const newStatus = approved ? PurchaseRequestStatus.APPROVED : PurchaseRequestStatus.DENIED;
+      setStatus(newStatus);
+      purchaseRequest.status = newStatus;
+      reviewPurchaseRequest(purchaseRequest, approved, userId);
+    }
```

Comment:
Pasting the comment I made in `PurchaseRequest` since it applies here too

>I don't think we should use isOnline here as the indicator when seeing if the purchase request has an offlineID seems to be the real limiting factor. We can execute the reviewPurchaseRequest so long as the purchaseRequest doesn't have an offline ID.

>With the current workflow, the User isn't able to approve any purchase requests while offline, when I think we really only need to prohibit them from reviewing their own purchase request that they might've made when they were offline